src/Client.php

```diff
@@ -338,6 +361,19 @@ public function previewTemplate( $templateId, $personalisation){
         return $this->httpPost( $path, $payload );
     }
 
+    /**
+     * Prepare a file before adding it to the $personalisation array for the sendEmail function
+     *
+     * @param string $file_contents
+     *
+     * @return string
+     */
+    public function prepareUpload( $file_contents ){
```

Comment:
This should check that the contents don't exceed the 2MB limit. We've decided to do this on the clients to make sure users always see the right message.